tests/test_stats.py

```diff
@@ -88,6 +88,13 @@ def test_gen_series_key_multiple_tags(self):
         expected = "my_measurement,alpha_tag=xyz,beta=aaa,mytag1=abcd,mytag2=defg,num_tag=123"
         self.assertEqual(series_key, expected)
 
+    def test_gen_series_key_special_chars_in_tag(self):
```

Comment:
@algrebe This is useful as a way to showcase examples. Please treat my other comment as a general point (preferable to co-locate example with the code).